galaxy_ng/app/management/commands/maintain-pe-group.py

```diff
@@ -29,7 +29,7 @@ def handle(self, *args, **options):
             'galaxy.group_admin',
             'galaxy.user_admin',
             'galaxy.collection_admin',
-            'galaxy.namespace_owner',
+            'galaxy.collection_namespace_owner',
             'galaxy.content_admin',
```

Comment:
`galaxy.collection_namespace_owner` can be removed, it's perms are covered in `galaxy.collection_admin`

`galaxy.content_admin` can be removed, it has perms that currently don't apply to the users in this group.

Reply:
if `signature_admin` gets added as a locked role, that should also be added here